src/main/resources/assets/plethora/lang/nl_be.lang

```diff
@@ -0,0 +1,148 @@
+
+item.plethora.module.module_scanner.name=Blokscanner
+item.plethora.module.module_scanner.adjective=Scanner
+item.plethora.module.module_scanner.desc=Staat toe dat blokken worden gescand in de buurt
+
+item.plethora.module.module_sensor.name=Entiteitsensor
+item.plethora.module.module_sensor.adjective=Entiteitsensor
+item.plethora.module.module_sensor.desc=Detecteert nabije entiteiten en verzamelt gegevens over hen
+
+item.plethora.module.module_kinetic.name=Kinetische uitbreiding
+item.plethora.module.module_kinetic.adjective=Kinetisch
+item.plethora.module.module_kinetic.desc=Maakt controle over je lichaam mogelijk via de neurale koppeling
+
+item.plethora.module.module_chat.name=Chatrecorder
+item.plethora.module.module_chat.adjective=Luisteren
+item.plethora.module.module_chat.desc=Maakt het luisteren van chat-gebeurtenissen mogelijk
+
+item.plethora.module.module_chat_creative.name=Creative chatrecorder
+item.plethora.module.module_chat_creative.adjective=Onderscheppen
+item.plethora.module.module_chat_creative.desc=Maakt het luisteren en onderscheppen van chat-gebeurtenissen mogelijk
+
+item.plethora.module.module_glasses.name=Overlay-bril
+item.plethora.module.module_glasses.desc=Tekent een overlay over de wereld.
+
+item.plethora.module.module_unknown.name=Onbekende module
+item.plethora.module.module_unknown.desc=Wat heb je deze keer gebroken?
+
+# Manipulator
+tile.plethora.manipulator.name=Manipulator
+tile.plethora.manipulator.mark_1.name=Manipulator Mk I
+tile.plethora.manipulator.mark_2.name=Manipulator Mk II
+tile.plethora.manipulator.mark_1.desc=Wordt gebruikt voor te koppelen met een module vanaf een computer
+tile.plethora.manipulator.mark_2.desc=Wordt gebruikt voor te koppelen met meerdere modules vanaf een computer
+
+# Keyboard
+item.plethora.keyboard.name=Toetsenbord
+item.plethora.keyboard.adjective=Interactief
+item.plethora.keyboard.desc=Voer van op afstand in op computers. Shift+Rechts-click om te linken met een computer.
+item.plethora.keyboard.bound=Verbonden met blok
+item.plethora.keyboard.cleared=Verbinding vrijgemaakt
+item.plethora.keyboard.binding=Verbonden met %s
+item.plethora.keyboard.broken=Verbinding met %s verbroken
+item.plethora.keyboard.close=Toetsenbord geopend. Duw op ESC om te sluiten.
+
+# Neural interface/connector
+item.plethora.neuralConnector.name=Neurale connector
+item.plethora.neuralConnector.desc=Wordt gebruikt om verbinding te maken met neurale koppelstukken
+
+item.plethora.neuralInterface.name=Neuraal koppelstuk
+item.plethora.neuralInterface.desc=Gaat in je helm slot. Maakt verbinding met de neurale connector.
+
+gui.plethora.neuralInterface.peripherals=Bekijk randapparatuur
+gui.plethora.neuralInterface.modules=Bekijk modules
+
+# Redstone integrator
+tile.plethora.redstone_integrator.name=Redstone-integrator
+tile.plethora.redstone_integrator.desc=Redstone-niveaus ophalen en instellen voor deze blok
+
+# Upgrade adjectives
+adjective.plethora.clock=Chronal
```

Comment:
"Chronal" isn't Dutch
This should either be "Klok" or "Interval" depending on the meaning of clock in this case.

Reply:
Was literally translated from `Chronal` to `Chronisch` in aba8ec842f78f519d50b7fedf94c1c09f257f9ed, but that might indeed be the wrong word. @SquidDev where is this text used? I'm not sure what would be correct, and would need the context.